### ISSUE TITLE
Google home device not deleted if a device is renamed in google home

### ISSUE THREAD

**Issue by skank01** [leikoilja/ha-google-home#184]
## Describe the bug

After renaming a google home device in google home (the device switched room too) , the new device/entities are created by the addon, but the old ones stay present.
Even after reloading the addon....

## Version of the integration

Latest version

**Comment by KapJI** [comment]
This is not implemented. You can remove old entities manually in HA entity manager.

**Comment by skank01** [comment]
No you cant, you can only disable it.

**Comment by KapJI** [comment]
You may need to restart HA. Then it should be possible in entities registry:
<img width="594" alt="Screenshot 2021-04-12 at 19 31 41" src="https://user-images.githubusercontent.com/3767331/114443538-b6a41000-9bc5-11eb-97b9-d9c366247b6c.png">

**Comment by DurgNomis-drol** [comment]
Will be low priority as you can delete entities in HA directly as @KapJI have said 😊 Also adding auto delete, we will risc that devices will appear and be removed and appear again and so on.

**Comment by KapJI** [comment]
Right now we use `device_name` in entity unique id. I think glocaltokens can provide some real device specific id for the device. Then HA should handle renames itself.

**Comment by DurgNomis-drol** [comment]
Let's create a separat feature for changing the unique id from `device_name` to a "Real" unique id. 😃 

We can probably find a unique id from [this](https://rithvikvibhu.github.io/GHLocalApi/#operation/EurekaInfo) endpoint.

**Comment by leikoilja** [comment]
We can indeed either extract something from `eureka` endpoint or simply generate `uuid` on `glocaltokens` side :)

**Comment by leikoilja** [comment]
Now that we use `unique_id`(https://github.com/leikoilja/ha-google-home/releases/tag/v1.7.0) for devices this feature should be fixed. But needs to be double-checked, before we can close the issue
